controllers/actions.github.com/resourcebuilder.go

```diff
@@ -337,6 +399,7 @@ func (b *resourceBuilder) newEphemeralRunnerPod(ctx context.Context, runner *v1a
 	for k, v := range runner.Spec.PodTemplateSpec.Labels {
 		labels[k] = v
 	}
+	labels["actions-ephemeral-runner"] = string(corev1.ConditionTrue)
```

Comment:
do you want to move this to some constant you created?

Reply:
This label is used only in a single place, so I did not want to have it as a constant.
There are other labels similar to this and I left them as string literals